recipe/meta.yaml

```diff
@@ -15,10 +15,18 @@ build:
     - /usr/lib/libresolv.9.dylib  # [osx]
     - /usr/lib64/libgcc_s.so.1  # [linux]
   script:
+    - export PYO3_CROSS_LIB_DIR={{ PREFIX }}/lib  # [arm64]
+    - find {{ PREFIX }}/lib/python3.* -name "_sysconfigdata_*.py" -type f -not -name "_sysconfigdata_arm64_*.py" -delete  # [arm64]
     - {{ PYTHON }} -m pip install . -vv
+  skip: True  # [arm64 and py==39]
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython                                 # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - pybind11                               # [build_platform != target_platform]
```

Comment:
Do these dependencies need to be added to build? The ['official' migration](https://github.com/conda-forge/tokenizers-feedstock/pull/19) doesn't do this

Reply:
I don't think it actually is. I natively copied from here: https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/#how-to-add-a-osx-arm64-build-to-a-feedstock

Also, I think that #19 does not have the Python 3.9 issue, so let's see whether removing these lines resolves the issue.